packages/hebao_v1/contracts/modules/core/WalletFactory.sol

```diff
@@ -80,8 +80,9 @@ contract WalletFactory is MetaTxAware
         external
         txAwareHashNotAllowed()
     {
+        address _walletImplementation = walletImplementation;
```

Comment:
I tested it and I am surprised the optimizer doesn't even optimize this one.

Reply:
The optimizer certainly isn't as good as you may expect from optimizers in other languages.